### ISSUE TITLE
sel4: build errors with gcc 10

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#4121]
When building sel4 with gcc 10, the following errors occur:

```
 [CC] kernel_final.s
/.../contrib/sel4-7935487f91a31c0cd8aaf09278f6312af56bb935/src/kernel/sel4/src/arch/x86/kernel/vspace.c: In function ‘init_vm_state’:
/.../contrib/sel4-7935487f91a31c0cd8aaf09278f6312af56bb935/src/kernel/sel4/src/arch/x86/kernel/vspace.c:498:14: error: taking address of packed member of ‘struct <anonymous>’ may result in an unaligned pointer value [-Werror=address-of-packed-member]
  498 |     init_tss(&x86KSGlobalState[CURRENT_CPU_INDEX()].x86KStss.tss);
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/.../contrib/sel4-7935487f91a31c0cd8aaf09278f6312af56bb935/src/kernel/sel4/src/arch/x86/kernel/vspace.c:499:63: error: taking address of packed member of ‘struct <anonymous>’ may result in an unaligned pointer value [-Werror=address-of-packed-member]
  499 |     init_gdt(x86KSGlobalState[CURRENT_CPU_INDEX()].x86KSgdt, &x86KSGlobalState[CURRENT_CPU_INDEX()].x86KStss.tss);
      |                                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by cproc** [comment]
Fixed in 63ef13f

**Comment by nfeske** [comment]
Merged to staging.
